packages/curve-ui-kit/src/features/connect-wallet/lib/wagmi/adapter.ts

```diff
@@ -12,13 +12,6 @@ export function clientToProvider(client: Client<Transport, Chain>) {
     name: chain.name,
     ensAddress: chain.contracts?.ensRegistry?.address,
   }
-  // todo: handle fallback providers, they need to be converted to I
```

Comment:
We should use the fallback provider ultimately right @0xAlunara ?

Reply:
Wasn't sure, just saw it was commented out. I guess so if the docs say so